src/services/types.ts

```diff
@@ -2,6 +2,7 @@ export type Category = {
   id: string;
   name: string;
   slug: string;
+  image: string;
```

Comment:
I think here we should have

`image: string | null`

Because in the DB the column is not REQUIRED, thus it can have missing values for images. So in the UI those will come as `null` right?